components/kubeflow/launcher/Dockerfile

```diff
@@ -11,12 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM ubuntu:16.04
```

Comment:
What image version we will be using with this new change?

Reply:
python:3.6, it's on line 15.

Reply:
I see, the current compiler version is python:3.7, should we keep both python versions consistent?

Reference: https://github.com/kubeflow/pipelines/blob/master/backend/Dockerfile#L11